File: 문자열 다루기 기본.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <string.h>
//0 = 48 1 = 49 9 = 57
bool solution(const char* s) {
    if (strlen(s) != 4 && strlen(s) != 6) {
        return false;
    }
    
    for (int i = 0; i < strlen(s); i++){
        if (s[i] > 57){
            return false;
        }
    }
    
    return true;
}

File: 문자열 다루기 기본.py
"""
def solution(s):
    return s.isnumeric()
"""
#2022.11.12
import re

def solution(s):
    if len(s) == 4 or len(s) == 6:
        if re.search('[a-zA-Z]', s):
            return False
        return True
    return False
